server/lib/nl/fulfillment/utils.py

```diff
@@ -213,11 +217,24 @@ def is_coplottable(chart_vars: ChartVars, places: List[Place]) -> bool:
   # Ensure all SVs have the same unit.
   unit = None
   for i, sv in enumerate(svs):
+    sv_exist_checks = exist_checks.get(sv, {})
     for place in places:
-      u = chart_vars.sv_exist_facet.get(sv, {}).get(place.dcid,
-                                                    {}).get('unit', '')
+      u = sv_exist_checks.get(place.dcid, ExistInfo()).facet.get('unit', '')
       if i > 0 and unit != u:
         return False
       unit = u
 
   return True
+
+
+# For a list of svs and places, Gets a map of sv -> place -> facetId from
+# the results of an existence check
+def get_sv_place_facet_ids(svs: List[str], places: List[Place],
+                           exist_checks: Dict[str, Dict[str, ExistInfo]]):
+  sv_place_facet_ids = {}
+  for sv in svs:
+    sv_place_facet_ids[sv] = {}
+    for pl in places:
+      sv_place_facet_ids[sv][pl.dcid] = exist_checks.get(sv, {}).get(
+          pl.dcid, ExistInfo()).facet.get('facetId', '')
+  return sv_place_facet_ids
```

Comment:
Can this be in https://github.com/datacommonsorg/website/blob/e48cbfa692be3b6d458f2938dbab74490ef27f8a/server/lib/explore/existence.py#L4?

Reply:
good idea, done